addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/env-type-config-service.js

```diff
@@ -241,6 +240,33 @@ class EnvTypeConfigService extends Service {
     return savedConfig;
   }
 
+  async checkAndUpdateParams(envType, config) {
+    const isAppStreamEnabled = this.settings.getBoolean(settingKeys.isAppStreamEnabled);
+    const updatedConfig = { ...config };
+    updatedConfig.params.push({
+      key: 'IsAppStreamEnabled',
+      value: isAppStreamEnabled.toString(),
+    });
+    let params = [...updatedConfig.params];
+    if (isAppStreamEnabled) {
+      params.push({
+        key: 'AccessFromCIDRBlock',
+        value: '',
+      });
+    } else {
```

Comment:
Yeah, that's what I meant. I think all the templates have a default value for `AccessFromCIDRBlock` so the validation shouldn't fail. I approved your change since this seemed a minor point.